libr/bin/p/bin_p9.c

```diff
@@ -416,30 +666,10 @@ static ut64 size(RBinFile *bf) {
 
 #if !R_BIN_P9
 
-/* inspired in http://www.phreedom.org/solar/code/tinype/tiny.97/tiny.asm */
```

Comment:
dont remove the create method pls. but it will be good to extend it to support something else than i386